pennylane/plugins/default_qubit.py

```diff
@@ -470,16 +484,29 @@ def _get_operator_matrix(self, operation, par):
             return A
         return A(*par)
 
+    def _get_tensor_operator_matrix(self, obs, par):
+        """Get the operator matrix for a given tensor of operations.
+
+        Args:
+            obs (list[str]): list of observable names to tensor
+            par (list[list[Any]]): parameter values
+
+        Returns:
+            array: matrix representation.
+        """
+        ops = [self._get_operator_matrix(o, p) for o, p in zip(obs, par)]
+        return functools.reduce(np.kron, ops)
```

Comment:
cool, how come you went with `reduce`?

Reply:
In my heart I'm a functional programmer :laughing: How would you have done it?

Reply:
:man_shrugging: